Lib/fontTools/misc/roundTools.py

```diff
@@ -0,0 +1,59 @@
+"""
+Various round-to-integer helpers.
+"""
+
+from fontTools.misc.py23 import *
```

Comment:
Would be fantastic if importing py23 could be avoided here.

Reply:
Oh, didn't know we are avoiding imports in new modules. Cool. Done.